.gitlab-ci.yml

```diff
@@ -17,6 +17,8 @@
 variables:
   DOCKER_TLS_CERTDIR: ""
   DOCKER_IMAGE: ${CI_REGISTRY_IMAGE}:${CI_COMMIT_SHORT_SHA}
+  NFS_SERVER: ${HL_NFS_SERVER_CODON}
+  NFS_PATH: ${HL_NFS_PATH_CODON}
```

Comment:
The variable which will be exposed to runner is `HL_NFS_OPPATH_CODON`. 
Instead of  `HL_NFS_PATH_CODON` it should be `HL_NFS_OPPATH_CODON`

Reply:
Updated